### PR TITLE
fix(network): align HTTP status codes with IANA

### PR DESCRIPTION
**Pull request by aslushnikov**
Aline HTTP status texts with https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml

### CODE REVIEW

**Comment by JoelEinbinder** [comment]
They are missing some that we have, like I’m a teapot. I feel like we are entering a scary world.

**Comment by aslushnikov** [comment]
> They are missing some that we have, like I’m a teapot. I feel like we are entering a scary world.

These are added

File: lib/NetworkManager.js [hunk 1]
```diff
@@ -773,9 +776,12 @@ const STATUS_TEXTS = {
   '422': 'Unprocessable Entity',
   '423': 'Locked',
   '424': 'Failed Dependency',
+  '425': 'Too Early',
   '426': 'Upgrade Required',
+  '427': 'Unassigned',
   '428': 'Precondition Required',
   '429': 'Too Many Requests',
+  '430': 'Unassigned',
```
Comment (JoelEinbinder) [inline lib/NetworkManager.js]: unassigned just means unassigned, its not a specific code.

**Review by JoelEinbinder — APPROVED** [review]

File: lib/NetworkManager.js (hunk 1 above)
Comment (aslushnikov) [reply lib/NetworkManager.js]: indeed!

**Pull request merged**